docs/reference/commandline/tag.md

```diff
@@ -10,11 +10,23 @@ parent = "smn_cli"
 
 # tag
 
-    Usage: docker tag [OPTIONS] IMAGE[:TAG] [REGISTRYHOST/][USERNAME/]NAME[:TAG]
+    Usage: docker tag [OPTIONS] NAME[:TAG] NAME[:TAG]
 
     Tag an image into a repository
 
       --help               Print usage
 
+An image name is made up of slash-separated name components, optionally prefixed
+by a registry hostname. The hostname must comply with standard DNS rules, but
+may not contain underscores. If a hostname is present, it may optionally be
+followed by a port number in the format `:8080`. Name components may contain
+lowercase characters, digits and separators. A separator is defined as a period,
+one or two underscores, or one or more dashes. A name component may not start or
+end with a separator.
+
+A tag name may contain lowercase and uppercase characters, digits, underscores,
+periods and dashes. A tag name may not start with a period or a dash and may
+contain a maximum of 128 characters.
+
```

Comment:
looks like there are some examples in the man/docker-tag.1.md file, we should have the same information in the online documentation and the man pages (examples there may need a refresh though)